install/image-references

```diff
@@ -38,10 +38,6 @@ spec:
     from:
       kind: DockerImage
       name: registry.svc.ci.openshift.org/openshift:ironic
-  - name: ironic-inspector
-    from:
-      kind: DockerImage
-      name: registry.svc.ci.openshift.org/openshift:ironic-inspector
```

Comment:
@iurygregory I think that also the other Ironic related images references could be removed, since they have been moved in [CBO](https://github.com/openshift/cluster-baremetal-operator/blob/master/manifests/image-references).

cc @sadasu @kirankt

Reply:
I think we can remo all of them (ironic, ironic-ipa-downloader, ironic-machine-os-downloader  and ironic-static-ip-manager)  @sadasu  do you think it makes sense to remove all or MAO would need something to work with CBO?